plugins/git/README.md

```diff
@@ -120,6 +121,8 @@ plugins=(... git)
 | gp                   | git push                                                                                                                                                                                 |
 | gpd                  | git push --dry-run                                                                                                                                                                       |
 | gpf                  | git push --force-with-lease                                                                                                                                                              |
```

Comment:
Should have replaced this line, not added to

Reply:
Fixed, thanks for the review!